llama-index-integrations/indices/llama-index-indices-managed-postgresml/llama_index/indices/managed/postgresml/query.py

```diff
@@ -106,29 +119,22 @@ def with_retriever(self, retriever: PostgresMLRetriever) -> "PostgresMLQueryEngi
             retriever=retriever,
         )
 
-    def _query(self, query_bundle: QueryBundle, **kwargs) -> RESPONSE_TYPE:
+    def _query(self, query_bundle: QueryBundle) -> RESPONSE_TYPE:
         """Answer a query."""
         if self._streaming:
-            async_token_gen = run_async_tasks([self._do_query(query_bundle, **kwargs)])[
-                0
-            ]
+            async_token_gen = run_async_tasks([self._do_query(query_bundle)])[0]
             return StreamingResponse(response_gen=async_token_gen.response_gen)
         else:
-            return run_async_tasks([self._do_query(query_bundle, **kwargs)])[0]
+            return run_async_tasks([self._do_query(query_bundle)])[0]
 
     async def _aquery(self, query_bundle: QueryBundle, **kwargs) -> RESPONSE_TYPE:
         """Answer an async query."""
         return await self._do_query(query_bundle, **kwargs)
 
     async def _do_query(
-        self,
-        query_bundle: Optional[QueryBundle] = None,
-        query: Optional[Dict[str, Any]] = None,
-        vector_search_limit: Optional[int] = 4,
-        vector_search_document: Optional[Dict[str, Any]] = {"keys": ["id", "metadata"]},
-        model: Optional[str] = "meta-llama/Meta-Llama-3-8B-Instruct",
-        model_parameters: Optional[Dict[str, Any]] = {"max_tokens": 2048},
+        self, query_bundle: Optional[QueryBundle] = None
     ) -> RESPONSE_TYPE:
+        query = self._query_query
```

Comment:
is this even used? Trying to follow the logic 😅 It feels kind of weird to attach the query as a class attribute no?

Reply:
Yes, this has the same purpose as the one outlined below. I also renamed this one to `pgml_query` which I think makes more sense.